keps/sig-instrumentation/1933-secret-logging-static-analysis/kep.yaml

```diff
@@ -0,0 +1,39 @@
+title: Defend against accidental credential logging via static analysis interation into Prow.
+kep-number: "1933"
+authors:
+- "@PurelyApplied"
+owning-sig: sig-instrumentation
+participating-sigs:
+- sig-instrumentation
+- sig-testing
+status: implementable
+creation-date: "2020-08-12"
+reviewers:
+- "@BenTheElder"
+- "@dashpole"
+- "@ehashman"
+- "@logicalhan"
+- "@mikejoh"
+approvers:
+- TBD
```

Comment:
You may add me as an approver.

Reply:
I didn't know if that implied approval and didn't want to presume.  Updated!